projects/baby_driver/scripts/gpio_set.py

```diff
@@ -0,0 +1,32 @@
+"""This module provides the Python implementation to set GPIO pins using the CAN protocol"""
+
+import can_util
+
+def gpio_set(port, pin, state):
+    """
+    Sets a GPIO pin
+    Args:
+        port: port of the GPIO pin to set
+        pin: pin number of the GPIO pin to set
+        state: 0 if setting low, 1 if setting high
+    Returns:
+        A boolean value. If true, the CAN message was sent successfully, otherwise
+        a ValueError or TimeoutError was raised.
```

Comment:
I'd prefer this not return anything and raise an exception if something goes wrong - that means you don't need the try/except around the function calls.